mne/connectivity/spectral.py

```diff
@@ -140,7 +140,30 @@ def compute_con(self, con_idx, n_epochs):
         plv = np.abs(self._acc / n_epochs)
         self.con_scores[con_idx] = plv
 
+class _ciPLVEst(_EpochMeanConEstBase):
+    """corrected imaginary PLV Estimator."""
 
+    name = 'ciPLV'
+
+    def __init__(self, n_cons, n_freqs, n_times):
+        super(_ciPLVEst, self).__init__(n_cons, n_freqs, n_times)
+
+        # allocate accumulator
+        self._acc = np.zeros(self.csd_shape, dtype=np.complex128)
+
+    def accumulate(self, con_idx, csd_xy):
+        """Accumulate some connections."""
+        self._acc[con_idx] += csd_xy / np.abs(csd_xy)
+
+    def compute_con(self, con_idx, n_epochs):
+        """Compute final con. score for some connections."""
+        if self.con_scores is None:
+            self.con_scores = np.zeros(self.csd_shape)
+        imag_plv = np.abs(np.imag(self._acc)) / n_epochs
+        real_plv = np.real(self._acc) / n_epochs
+        corrected_imag_plv = imag_plv / np.sqrt(1 - real_plv ** 2)
```

Comment:
This is going to explode in cases where `real_plv == 1.` (purely real). Should we at least avoid the division by zero?